src/Bowling.js

```diff
@@ -0,0 +1,163 @@
+console.log("testtest");
+function BowlingGame() {
+  this.framesLeft = 10;
+  this.STRIKE = 10;
+  this.allScores = [];
+
+  this.currentRoll = 0;
+  this.currentScore = 0;
+  this.totalScore = null;
+
+  this.isStrike = false;
+  this.strikeCount = 0;
+  this.strikeScore = 0;
+  this.consecutiveStrikes = 0;
+
+  this.isSpare = false;
+  this.spareCount = 0;
+  this.spareScore = 0;
+
+  this.lastFrameExtra = false;
+
+  this.GAMEOVER_ERROR = "The game is over you cannot bowl again";
+
+  };
+
+  BowlingGame.prototype.bowl = function(pins) {
```

Comment:
This function has too many statements. (11)<br>This function's cyclomatic complexity is too high. (6)